src/main/java/com/faforever/moderatorclient/ui/main_window/TutorialController.java

```diff
@@ -0,0 +1,164 @@
+package com.faforever.moderatorclient.ui.main_window;
+
+import com.faforever.moderatorclient.api.domain.TutorialService;
+import com.faforever.moderatorclient.api.domain.events.MessagesChangedEvent;
+import com.faforever.moderatorclient.ui.*;
+import com.faforever.moderatorclient.ui.domain.TutorialCategoryFX;
+import com.faforever.moderatorclient.ui.domain.TutorialFx;
+import javafx.application.Platform;
+import javafx.beans.value.WeakChangeListener;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import javafx.collections.transformation.FilteredList;
+import javafx.collections.transformation.SortedList;
+import javafx.fxml.FXML;
+import javafx.scene.Node;
+import javafx.scene.Scene;
+import javafx.scene.control.CheckBox;
+import javafx.scene.control.SplitPane;
+import javafx.scene.control.TableView;
+import javafx.stage.Stage;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+import java.util.HashMap;
+
+
+@Slf4j
+@Component
+public class TutorialController implements Controller<Node> {
+    private final UiService uiService;
+    private final TutorialService tutorialService;
+    private final ObservableList<TutorialFx> tutorialList = FXCollections.observableArrayList();
+    private final FilteredList<TutorialFx> filterTutorials;
+    public SplitPane root;
+    public TableView<TutorialFx> tutorialTableView;
+    public TableView<TutorialCategoryFX> categoryTableView;
+    public CheckBox filterByCategoryCheckBox;
+    private HashMap<TutorialFx,WeakChangeListener<Boolean>> weakChangeListenersByTutorial = new HashMap<>();
+
+
+    @Inject
+    public TutorialController(UiService uiService, TutorialService tutorialService) {
+        this.uiService = uiService;
+        this.tutorialService = tutorialService;
+        filterTutorials = new FilteredList(tutorialList);
+    }
+
+    @FXML
+    public void initialize(){
+        Runnable refreshTutorials = this::onRefreshTutorials;
+        Runnable refreshCategorys = this::onRefreshCategorys;
+        ViewHelper.buildTutorialTable(tutorialTableView, tutorialService, log, refreshTutorials);
+        ViewHelper.buildCategoryTable(categoryTableView, tutorialService, refreshCategorys);
+        onRefreshTutorials();
```

Comment:
Same as above. Don't load data in @FXML initializers. Call them manually from MainController.